docker/k8s/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM sammyne/hyperledger-tcf-vm:alpha 
+
+LABEL maintainer=lixiangmin01@baidu.com
+
+RUN apt update -y && apt install -y vim
+# the sed cmd below is to decrease the 1TB LMDB cache size for local development
+RUN cd /project && rm -rf TrustedComputeFramework &&\
+    git clone --branch master --depth 1 https://github.com/hyperledger-labs/trusted-compute-framework.git &&\
+    mv trusted-compute-framework TrustedComputeFramework &&\
+    sed -i 's/^StorageSize.*/StorageSize = "4 GB"/g' TrustedComputeFramework/config/tcs_config.toml &&\
```

Comment:
Why do we need this? why don't we change the tcs_config file?

Reply:
This is to address the problem reported by PR https://github.com/hyperledger-labs/trusted-compute-framework/issues/19, where too large LMDB storage may cause booting failure